docs/docs/usage/worksheet-mode.md

```diff
@@ -24,27 +25,20 @@ println("Hello, worksheets!")
 456 + xyz
 ```
 
-On top of the buffer, the message `Run this worksheet` appears. Click it to
-evaluate the code of the worksheet. Each line of output is printed on the right
+On top of the buffer, the message `Copy Worksheet Output` appears. Save the file to evaluate the code of the worksheet. Each line of output is printed on the right
```

Comment:
I wouldn't mention `Copy Worksheet Output` here, it can be mentioned additionally.